packages/datadog_flutter_plugin/lib/src/rum/ddrum_web.dart

```diff
@@ -113,6 +113,11 @@ class DdRumWeb extends DdRumPlatform {
     DD_RUM?.addTiming(name);
   }
 
+  @override
+  Future<void> addViewLoadingTime(bool overwrite) async {
+    // NOOP - Not supported by the Browser SDK
```

Comment:
maybe it makes sense to make a log entry in such cases?

Reply:
The support matrix for web is pretty shallow, if I added a log here I'd be adding it for a lot of features that we don't support.

I think once more things are supported out of the box, I'll revisit and add more logs to things we don't support.